.travis.yml

```diff
@@ -46,6 +51,6 @@ deploy:
     file_glob: true
     file: "netdata*.tar.*"
     on:
-      condition: $CC = gcc
+      condition: $CC = gcc && $TRAVIS_OS_NAME
```

Comment:
thanks! it is already fixed: https://github.com/ktsaou/netdata/commit/2f756fa58aba243cd6579270ff93de28c09db1e5